### DISCUSSION TITLE
Run code before onDragStart

### DISCUSSION THREAD

**Discussion by yousif-khalil-abdulkarim** [clauderic/dnd-kit#1335] [Q&A]
I have sortable list that contains accordion items and I want to close the accordion items before drag start. So I wonder if it is possible to 
run code before onDragStart in other words before the dimension for sortable list items are captured.

[Link to codesandbox](https://codesandbox.io/p/sandbox/react-dnd-on-before-drag-start-y3g74g?file=%2Fsrc%2FApp.tsx%3A1%2C35)

**Comment by heyLukey** [comment]
Anyone find a solution for this?
